pkg/graphviz/traceflow.go

```diff
@@ -177,8 +177,14 @@ func getTraceflowMessage(o *opsv1alpha1.Observation) string {
 	if o.Component == opsv1alpha1.NetworkPolicy && len(o.NetworkPolicy) > 0 {
 		str += "\nNetpol: " + o.NetworkPolicy
 	}
+	if len(o.Pod) > 0 {
+		str += "\nTo: " + o.Pod
+	}
+	if o.Action != opsv1alpha1.Dropped && len(o.TranslatedDstIP) > 0 {
+		str += "\nIP: " + o.TranslatedDstIP
+	}
 	if o.Action != opsv1alpha1.Dropped && len(o.TunnelDstIP) > 0 {
-		str += "\nTo: " + o.TunnelDstIP
+		str += "\nDestination: " + o.TunnelDstIP
```

Comment:
Tunnel destination IP here may be better?

Reply:
`Tunnel Destination IP` and `Translated Destination IP` are good names in the above case, but may be a little bit long when pod name is short.
How about `Pod IP` for TranslatedDstIP, and `Node IP` for TranslatedDstIP?